scout/server/blueprints/variant/templates/variant/sv-variant.html

```diff
@@ -221,7 +221,7 @@
           <span class="d-inline">Copy number profile</span>
           <span class="d-inline ms-3">
             {% for sample in variant.samples %}
-              <a class="btn btn-secondary" target="_blank" href="http://{{gens_info.host}}/{{sample.display_name}}?region={{variant.chromosome}}:{{variant.position - 2000}}-{{variant.end + 2000}}&variant={{variant._id}}&genome_build={{gens_info.genome_build}}", {{variant.chromosome}})>Gens {{ sample.display_name }}</a>
+              <a class="btn btn-secondary" target="_blank" href="http://{{gens_info.host}}/{{sample.display_name}}?region={{variant.chromosome}}:{{variant.position - 2000}}-{{variant.end + 2000}}&variant={{variant._id}}&genome_build={{gens_info.genome_build}}&case_name={{case.case_id}}", {{variant.chromosome}})>Gens {{ sample.display_name }}</a>
```

Comment:
Will this change have an effect also on other gens instances different from ours?